docs/source/quantization/bnb.rst

```diff
@@ -0,0 +1,41 @@
+.. _bits_and_bytes:
+
+BitsAndBytes
+==================
+
+vLLM now supports `BitsAndBytes <https://github.com/TimDettmers/bitsandbytes>`_ for more efficient model inference.
+BitsAndBytes quantizes models to reduce memory usage and enhance performance without significantly sacrificing accuracy.
+This is particularly useful for deploying large language models in resource-constrained environments.
+Below are the steps to utilize BitsAndBytes with vLLM.
```

Comment:
Can you mention that it allows for 4bit compression without calibration data? I feel this is the biggest benefit of BNB

Reply:
ok